src/main/scala/contracts/token_sale_setup_proxy/v1.0.0/ergoscript/Token_Sale_Setup_Proxy.es

```diff
@@ -47,7 +48,7 @@
             }
 
             val validContract: Boolean = {
-                (tokenCollectionIssuerBoxOUT.propositionBytes == _TokenCollectionIssuerProxyContractBytes)
+                (tokenCollectionIssuerBoxOUT.propositionBytes == TokenCollectionIssuerProxyContractBytes)
```

Comment:
probably just a copypaste error, think this should be TokenCollectionIssuerContractBytes